consumer.go

```diff
@@ -24,8 +24,9 @@ type ConsumerOptions struct {
 	// Default: auto-detected from URL path.
 	Compression Compression
 
-	// OnError callback is triggered on sync errors.
-	OnError func(error)
+	// AfterSync callbacks are triggered after each sync, receiving
+	// the updated status and error (if occurred).
+	AfterSync func(updated bool, bool error)
```

Comment:
mistype - you meant "bool error"-> "err error"?

Reply:
Also, can it ever be `true, NonNilErr`?

Reply:
It returns:

* `true, nil` if the sync has updated the data
* `false, nil` if the sync has run but not updated any data (not modified)
* `false, err` if there was an error

Will fix typo

Reply:
I'd just have `ErrNotModified` for second case, probably

Reply:
@mxmCherry preferences ...